ui/apps/platform/src/Containers/Collections/CollectionResults.tsx

```diff
@@ -138,7 +138,9 @@ function CollectionResults({ dryRunConfig }: CollectionResultsProps) {
                                 variant="link"
                                 isInline
                                 className="pf-u-text-align-center"
-                                onClick={() => fetchDryRun(currentPage + 1)}
+                                onClick={() =>
+                                    fetchDryRun(dryRunConfig, currentPage + 1, filterText, selected)
```

Comment:
Off the topic of this pull request, seeing `currentPage + 1` reminds me of the following:

https://reactjs.org/docs/faq-state.html#what-is-the-difference-between-passing-an-object-or-a-function-in-setstate

* object `this.setState({ count: this.state.count + 1})`
* function
  ```js
  this.setState((state) => {
    // Important: read `state` instead of `this.state` when updating.
    return {count: state.count + 1}
  });
  ```

And also discussion how to render reloading state. Independent of other details about visual presentation, what is pro and con of a `disabled={isFetchingDryRun}` prop to make explicit that `currentPage` is a stable value?

Your viewpoint of the scenario is welcome if I am misreading or misunderstanding!

Reply:
I think that is definitely a good idea. In addition to the point about being a stable value, I think we would get into weird behavior if the user clicks the "view more" link multiple times before the request returns. I'll make a note to handle this as a follow up in the reloading state change, as well as hit the other places we are doing "view more" in the collections workflow.